### ISSUE TITLE
Versions text shows in sidebar instead of file

### ISSUE THREAD

**Issue by jdforsythe** [vscode-contrib/vscode-versionlens#23]
Extension v0.8.0

![versionlens](https://cloud.githubusercontent.com/assets/286972/20007246/becdec3a-a271-11e6-80d0-8ad7063f5d82.PNG)

As you can see, the text added by Version Lens shows up way to the left and slightly above where it should be - it's in the sidebar! Moving the `package.json` file to the second or third editor has no effect on the position of the text. Neither does changing the window or sidebar size. Restarting VS code also had no effect.

Just noticed this after the VS Code 1.7.1 update.

**Comment by pflannery** [comment]
@jdforsythe This looks like vscode  code lens bug. 
I cant recreate this on Windows 10 with VS Code 1.7.1. 
What OS are you using?

**Comment by jdforsythe** [comment]
@pflannery I'm on Windows 10 with VS Code 1.7.1.

**Comment by jdforsythe** [comment]
I tried disabling and re-enabling the extension to no avail. Not sure if it's related, but I was going to uninstall and reinstall it - and when I click uninstall this is thrown to the developer tools console:

```
shell.ts:440 Cannot read property 'id' of null: TypeError: Cannot read property 'id' of null
    at file:///C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:127:3158
    at Array.some (native)
    at e.getExtensionState (file:///C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:127:3115)
    at t.stateProvider (file:///C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:126:27788)
    at t.get [as state] (file:///C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:126:24311)
    at t.update (file:///C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:116:8410)
    at file:///C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:116:8105
    at e.invoke (file:///C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:29:22801)
    at e.fire (file:///C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:30:1364)
    at e.onUninstallExtension (file:///C:/Program Files (x86)/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:127:2335)
```

This does not happen with other extensions (I've tried about 10 of them).

**Comment by pflannery** [comment]
@jdforsythe 
Looks like it's the zoom feature of vscode that is pushing the code lens out of position. See https://github.com/Microsoft/vscode/issues/14980

I can't recreate the "'id' of null" error. Instead it uninstalls as expected
Could you try remove the folder manually at `C:\Users\YOURUSERNAME\.vscode\extensions\pflannery.vscode-versionlens-0.8.0`

**Comment by jdforsythe** [comment]
@pflannery I was able to remove and reinstall. I can't recreate the error on clicking uninstall, either. Some sort of fluke.

And nice find on the zoom. I reset zoom and the versions line up properly. The odd thing is that their position isn't affected by _what_ the zoom level is - they're always misplaced at the same spot regardless of whether you've zoomed in or out and to what level.

**Comment by pflannery** [comment]
this is fixed in 1.8.0-insider, see https://github.com/Microsoft/vscode/commit/504877fa3550eb8e4f1854dd17055e8fc8781fc3
